pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-headers.adoc

```diff
@@ -0,0 +1,125 @@
+= Reading and Writing to Request Headers and Body
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+PDK enables you to both:
+
+* <<read-and-write-headers>>
+* <<read-and-write-request-bodies>>
+
+[[read-and-write-headers]]
+== Read and Write Request Headers
+
+To access the headers in both the request and the response, transform the `RequestState` or `ResponseState` to a header state by calling the method `into_headers_state()` and awaiting it. After calling either method, access and manipulate the headers by calling the functions of the `HeadersHandler` trait.
```

Comment:
```suggestion
To access the headers in both the request and the response, transform the `RequestState` or `ResponseState` to a header state by calling the method `into_headers_state()` and awaiting it. After calling the method, access and manipulate the headers by calling the functions of the `HeadersHandler` trait.
```
You only mention one method in the previous sentence? Is there supposed to be another? "Either" implies two.